src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java

```diff
@@ -230,9 +230,11 @@ public void applyTripUpdates(final Graph graph, final boolean fullDataset, final
                     case CANCELED:
                         applied = handleCanceledTrip(tripUpdate, feedId, serviceDate);
                         break;
-                    case MODIFIED:
-                        applied = validateAndHandleModifiedTrip(graph, tripUpdate, feedId, serviceDate);
-                        break;
+// Evan Siroky note: MODIFIED doesn't seem to be a valid value after updating to the latest GTFS-RT bindings. Where did
+// this come from and is it still needed/or used anywhere?
+//                    case MODIFIED:
+//                        applied = validateAndHandleModifiedTrip(graph, tripUpdate, feedId, serviceDate);
+//                        break;
```

Comment:
Yea, this is confusing. I would say maybe we just remove entirely? I'm not really sure though. I did a very brief search and found `MODIFIED` mentioned in https://github.com/opentripplanner/OpenTripPlanner/issues/2780 with a link to to this coderef:

https://github.com/opentripplanner/OpenTripPlanner/blob/de5454238b1132e55e5f22e11c287e58f506a57d/src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java#L302-L305

Reply:
Well, this comment seems to explain things: https://github.com/ibi-group/OpenTripPlanner/blob/78639646c0ba61db359745215bce6df0ed30d399/src/main/java/com/google/transit/realtime/GtfsRealtime.java#L7-L25

The latest gtfs-rt-bindings include a deprecated field of REPLACEMENT with the same value, so I went ahead and changed things to use that in d801a8ffbd6db673458094a7ac9b9a6dbe92006e.